### PR TITLE
IDVA3-1663 Updated psc statement screen

### PR DESCRIPTION
**Pull request by JasSmith-ch**
- updated screen to match the v9 prototype
- updated the tests to match it

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: locales/en/individual-statement.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "individual_statement_title": "Confirm the identity verification statement",
     "individual_statement_value": "individual_statement",
     "individual_statement_part1": "I confirm that ",
-    "individual_statement_part2": " has verified their identity.",
-    "individual_statement_born": "Born "
+    "individual_statement_part2": " has verified their identity for Companies House.",
+    "individual_statement_born": "Born ",
+    "individual_statement_button": "Continue and submit"
```
Comment (eedwards0) [inline locales/en/individual-statement.json:7]: Please update to "Confirm and submit"

**Review by eedwards0 — CHANGES_REQUESTED** [review]
Button text requires updating

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by eedwards0 — APPROVED** [review]
Looks good :)

**Pull request merged**